### DISCUSSION TITLE
Route /docs/auth seems to be broken

### DISCUSSION THREAD

**Discussion by FireIsGood** [JanDeDobbeleer/oh-my-posh#5046] [Troubleshoot]
### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### What would you like to see changed/added?

The route `/docs/auth` from `website/docs/auth.mdx` seems to be broken and also unused.

When running `npm run build`, the build crashes trying to create the route and the route itself has no links in the source code's `sidebar.js`.

![image](https://github.com/JanDeDobbeleer/oh-my-posh/assets/109556932/9051af1b-896e-4076-8156-abd252535485)

On the [currently active site](https://ohmyposh.dev/docs/auth), this route also doesn't seem to be used correctly as there is no sidebar 

![image](https://github.com/JanDeDobbeleer/oh-my-posh/assets/109556932/51ca2f0d-6976-450d-9daa-341f0a5ba714)

I'm not sure what this segment is or if it's actually a feature, but as of right now it's broken and crashes the site with it on building.

**Comment by JanDeDobbeleer** [comment]
@FireIsGood I'm unable to reproduce this locally and neither can the [build](https://github.com/JanDeDobbeleer/oh-my-posh/actions/runs/9450185465/job/26028173784).

<img width="972" alt="image" src="https://github.com/JanDeDobbeleer/oh-my-posh/assets/2492783/c2c38cba-0e8d-4915-90e1-a16d5645b7af">

This page is used to reroute upon using oauth for [Strava](https://ohmyposh.dev/docs/segments/strava) and [Withings](https://ohmyposh.dev/docs/segments/withings#accessing-your-withings-data) so you can retrieve the tokens. The page also works on the [site](https://ohmyposh.dev/docs/auth).
